content/applications/marketing/marketing_automation.rst

```diff
@@ -1,17 +1,126 @@
+filter rule fields.
+
+In the rule fields, customizable equations can be configured for Odoo to use when filtering who to
+include or exclude in this specific marketing campaign.
+
+.. image:: marketing_automation/filter-node-equation-fields.png
+   :align: center
+   :alt: How the filter rule equation fields look in Odoo Marketing Automation campaigns.
+
+.. note::
+   :guilabel:`Records` refer to contacts in the system that fit the specified criteria for a
+   campaign.
+
+Also, once :guilabel:`Add condition` is clicked, the ability to :guilabel:`Save as Favorite Filter`
+becomes available on the campaign form.
+
+There is also the option to match records with :guilabel:`all` or :guilabel:`any` of the rules
+configured in the :guilabel:`Filter` field.
+
+To choose either of those options, click :guilabel:`all` from the middle of the sentence
+":guilabel:`Match records with all of the following rules`" to reveal a drop-down menu with those
+options.
+
+.. image:: marketing_automation/match-all-any-rules-drop-down.png
+   :align: center
+   :alt: Match records with all or any of the rules in Filter field for marketing campaigns.
+
+When the first field of the rule equation is clicked, a nested drop-down menu of options appears on
+the screen where specific criteria is chosen based on needs of the campaign.
+
+The remaining fields on the rule equation further define the criteria, which is used to determine
+which records in the database to include or exclude in the execution of the campaign.
+
+To add another rule, either click the :guilabel:`➕ (plus sign)` icon to the right of the filtering
+rule, or click :guilabel:`New Rule` beneath the rule equation fields. When either are clicked, a new
+series of rule fields appears.
+
+To add a branch of multiple rules at the same time, click the :guilabel:`branch` icon, located to
+the right of the :guilabel:`➕ (plus sign)` icon. When clicked, two additional sub-rule equation
+fields appear beneath the initial rule.
+
+.. image:: marketing_automation/rule-branch-filter-sample.png
+   :align: center
+   :alt: Sample of how the rule branches look in the filter section of a marketing campaign.
+
+There is also the option to have the filter apply to :guilabel:`any` or :guilabel:`all` of the
+configured branch rules.
+
+For further information on marketing automation campaign filter configuration, refer to the
+:doc:`marketing_automation/target_audience` documentation.
+
 .. seealso::
-   - `Odoo Tutorials: Marketing <https://www.odoo.com/slides/marketing-27>`_
+   - :doc:`marketing_automation/target_audience`
+   - :doc:`marketing_automation/workflow_activities`
+   - :doc:`marketing_automation/testing_running`
+   - :doc:`marketing_automation/advanced/understanding_metrics`
 
 .. toctree::
    :titlesonly:
 
-   marketing_automation/getting_started
+   marketing_automation/target_audience
+   marketing_automation/workflow_activities
+   marketing_automation/testing_running
    marketing_automation/advanced
```

Comment:
Do we think having a sub-directory called `advanced` is necessary?
Why not just have it at the same level as all the other docs as this also solves the toctree display issue?

What do you think?

Reply:
I agree, my plans for improving this documentation do not invlove an `/advanced` directory, so I am happy to remove this as well and move `understanding_metrics.rst` to the same level as the others 👍